tests/springboot/src/main/resources/hawtio-static/hawtconfig.json

```diff
@@ -3,8 +3,8 @@
     "appName": "Hawtio Spring Boot 2 Sample app for E2E testing purposes",
     "appLogoUrl": "img/hawtio-logo.svg",
     "companyLogoUrl": "img/hawtio-logo.svg",
```

Comment:
What about these logos @tadayosi?

Reply:
You can copy them from the console itself but unless you provide your own they just point to the builtin img resources.